<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInterfaceClass</class>
 <widget class="QMainWindow" name="UserInterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>629</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>ClusterWorker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="GroupBoxClientList">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>291</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Client List</string>
    </property>
    <widget class="QPushButton" name="ButtonKickClient">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>271</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Kick Client</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="TableClientList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>271</width>
       <height>221</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GroupBoxApplication">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Application</string>
    </property>
    <widget class="QPushButton" name="ButtonServerFileSelect">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonDeployApplication">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>241</width>
       <height>23</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="accessibleDescription">
      <string extracomment="Startet Das Programm mit den Parametern f端r das Cluster"/>
     </property>
     <property name="text">
      <string>Deploy Application</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="LabelClientFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Client File</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TextBoxClientFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="TextBoxServerFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonClientFileSelect">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
    <widget class="QLabel" name="ServerClientFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Server File</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GroupBoxMenu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>261</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QPushButton" name="ButtonAbort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="accessibleDescription">
      <string extracomment="Startet Das Programm mit den Parametern f端r das Cluster"/>
     </property>
     <property name="text">
      <string>Abort</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonStart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="accessibleDescription">
      <string extracomment="Startet Das Programm mit den Parametern f端r das Cluster"/>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QProgressBar" name="ProgressBarWork">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QGraphicsView" name="ImageStatus">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GroupBoxConsole">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>310</y>
      <width>291</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Console</string>
    </property>
    <widget class="QTextBrowser" name="TextConsole">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>271</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GroupBoxInit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TextBoxPort">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonOpenServer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>241</width>
       <height>23</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="accessibleDescription">
      <string extracomment="Startet Das Programm mit den Parametern f端r das Cluster"/>
     </property>
     <property name="text">
      <string>Open Server</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="LabelIPMode">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>IP Mode</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ComboBoxIPMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GroupBoxWork">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Work</string>
    </property>
    <widget class="QPushButton" name="ButtonWorkFileSelect">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TextBoxWorkFilePath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonDeployWork">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>241</width>
       <height>23</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="accessibleDescription">
      <string extracomment="Startet Das Programm mit den Parametern f端r das Cluster"/>
     </property>
     <property name="text">
      <string>Deploy Work</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="LabelWorkFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Work File</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="UserInterface.qrc"/>
 </resources>
 <connections/>
</ui>
